modules/common-attributes.adoc

```diff
@@ -61,9 +61,10 @@ endif::[]
 :product-registry: OpenShift image registry
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:rhacs-version: 4.9.1
+:rhacs-version: 4.9.2
 :ga-date-490: 30 October 2025
 :ga-date-491: 24 November 2025
+:ga-date-492: 17 December 2025
```

Comment:
We're changing this to December 16th after all, unless the konflux release pipeline (now running) fails and remediation takes more time. I will post another update once it completes.

Reply:
Release pipeline passed, please change this to:

```suggestion
:ga-date-492: 16 December 2025
```
Otherwise LGTM